README.md

```diff
@@ -35,7 +35,6 @@
 # 📖 [Movies] <a name="about-project"></a>
```

Comment:
- [ ] Kindly let's give a meaningful app title 👇🏻 and improve your project description 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/235334813-7e5a9679-cfd6-418b-b05b-f0d9d5183e3b.png)